man/systemd.network.xml

```diff
@@ -153,6 +153,14 @@
             </para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>SSID=</varname></term>
+          <listitem>
+            <para>A whitespace-separated list of shell-style globs matching the SSID of the currently
+            connected wireless network. If the list is prefixed with a "!", the test is inverted.
```

Comment:
wouldn't "Wi-Fi" be more correct than "wireless"? the latter sounds more generic to me, and may include things like bluetooth, cellular, etc.

Reply:
Quoting Wikipedia: 
> Wi‑Fi is a trademark of the Wi-Fi Alliance, [...]

One can't use the term "Wi-Fi" without violating their trademark as the linux kernel does not implement anything certified but only a subsystem that controls IEEE 802.11 devices. It is bad enough that the widespread marketing creeps this term into code and technical documents (e.g. the ProxyARPWiFi= config that is named after the kernel setting).

The linux kernel rightfully calls this cfg80211 subsystem and the correct terminology is either Wireless LAN or WLAN.

Reply:
Fair enough